lib/tasks/after_tasks.rake

```diff
@@ -631,6 +631,28 @@ namespace :After do
     end
     puts && STDOUT.flush
   end
+
+  desc "Add user_id to existing logged in kudos without user_ids"
+  task(add_user_id_to_kudos: :environment) do
+    kudos_to_update = Kudo.where("pseud_id IS NOT NULL AND user_id IS NULL")
+    kudos_count = kudos_to_update.count
+    total_batches = kudos_count / 1000
```

Comment:
It does, yeah. The math to correct it just seemed too hacky to be worth satisfying my own annoyance with it, so I went with saying "about X batches" instead. Since it's annoying other people, though, I can change it!